process_report/process_report.py

```diff
@@ -62,16 +65,45 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    try:
+        s3_resource = boto3.resource(
+            service_name="s3",
+            endpoint_url=os.environ.get(
+                "S3_ENDPOINT", "https://s3.us-east-005.backblazeb2.com"
+            ),
+            aws_access_key_id=os.environ["S3_KEY_ID"],
+            aws_secret_access_key=os.environ["S3_APP_KEY"],
+            config=Config(
+                signature_version="s3v4",
```

Comment:
Out of curiosity, which documentation pointed to setting this setting?

Reply:
I just copied the code from Backblaze's [sample code](https://github.com/backblaze-b2-samples/b2-python-s3-sample/blob/91d68ce06f7cb72d38c9b32253ff0c411480c959/sample.py#L118). 

Reading a bit more into their [documentation](https://www.backblaze.com/apidocs/introduction-to-the-s3-compatible-api), it seems they specify the signature version because their S3 API only supports authentication with v4 signatures. But it doesn't seem to matter for our current code since we're using the boto3 SDK, not the web API, as noted by the Amazon S3 [documentation](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html):
"If you use the AWS SDKs (see [Sample Code and Libraries](https://aws.amazon.com/code)) to send your requests, you don't need to read this section because the SDK clients authenticate your requests by using access keys that you provide."

I tested my code without specifying the signature version and it works normally. I now assume you want the config removed?